common/utils/iiif.ts

```diff
@@ -33,7 +34,35 @@ export function getAuthService(
   }
 }
 
-export function getImageAuthService(canvas?: IIIFCanvas) {
+export function getVideoClickthroughService(
+  video: IIIFMediaElement
+): AuthService | undefined {
+  if (video?.service) {
+    if (Array.isArray(video.service)) {
+      return video.service.find(
```

Comment:
Why do we sometimes have arrays?
Could we leave a note so futures devs will understand?